config/locales/sufia.en.yml

```diff
@@ -1,22 +1,22 @@
 en:
   sufia:
-    account_name:       "My Institution Account Id"
+    account_name:       "PID"
     institution_name:   "Virginia Tech"
     product_name:       "VTechData"
     account_label:      "User"
-    deposit_agreement:  "VTechData's Deposit Agreement"
+    deposit_agreement:  "VTechData Deposit Agreement"
     document_language:  "en"
     upload_tooltip:     "Please accept Deposit Agreement before you can upload."
-    share_button:       "Share Your Work"
+    share_button:       "Share Your Dataset"

```

Comment:
http://english.stackexchange.com/questions/2120/which-is-correct-dataset-or-data-set